index.bs

```diff
@@ -2527,6 +2530,31 @@ needs.
 </div>
 
 
+## Feature Policy integration ## {#sctn-feature-policy}
+
+This specification defines a [=policy-controlled feature=] identified by
+the feature policy token "<code><dfn data-lt="webauthn-feature">webauthn</dfn></code>".
+Its [=default allowlist=] is '<code>self</code>'. [[!Feature-Policy]]
+
+A {{Document}}'s [=Document/feature policy=] determines whether any content in that <a href="https://html.spec.whatwg.org/multipage/dom.html#documents">document</a> is allowed
+to successfully invoke the [=Web Authentication API=], i.e., via
```

Comment:
&ldquo;i.e.&rdquo; to &ldquo;e.g.&rdquo;, since there are other Webauthn calls (IsUVPAA) and there may be others in the future?

Reply:
@agl see below https://github.com/w3c/webauthn/pull/1214#issuecomment-491014211 wrt IsUVPAA().

Reply:
in any case, done.